invenio_records_files/__init__.py

```diff
@@ -61,31 +61,20 @@
 
 >>> from invenio_records_files.api import Record
 
-This ``Record`` class has a special property ``files`` through which you can
-access and create files. By default this property is ``None``:
+This :py:data:`~invenio_records_files.api.Record` class has a special property
+``files`` through which you can access and create files. By default the
+class creates a bucket when you create a bucket:
```

Comment:
Shouldn't this read: "By default, the class creates a bucket when you create a Record instance:" ?